<launch>
	<node pkg='turtlesim' type='turtlesim_node' name='simulator'/>
	<node pkg='turtlesim' type='turtle_teleop_key' name='keyboard'/>
	<node pkg='tf_study' type='turtle_tf_satellite.py' name='turtle1_tf_broadcaster' output='screen'>
		<remap from='input_pose' to='turtle1/pose'/>
		<param name='turtle_tf_name' type='string' value='turtle1'/>
	</node>
	<node pkg='tf' type='static_transform_publisher' name='turtle_2_carrot_tf' args='0.5 0 0 0 0 0 turtle1 carrot 10'/>
	<node pkg='rviz' type='rviz' name='rviz' args='-d $(find tf_study)/rviz/turtle_satellite.rviz' />
</launch>
